types/office-js/index.d.ts

```diff
@@ -1986,9 +1986,14 @@ declare namespace Office {
      *
      * @remarks
      *
-     * **Applications**: Excel, Outlook (in preview), PowerPoint, Word
-     *
-     * `OfficeTheme` is only supported in Office on Windows, Mac, and the web.
+     *  *Supported applications, by platform*
+     *  <table>
+     *   <tr><th>                             </th><th> Office on Windows </th><th> Office on the web </th></tr>
+     *   <tr><td><strong> Excel      </strong></td><td> Y                 </td><td>                   </td></tr>
+     *   <tr><td><strong> Outlook    </strong></td><td> Preview           </td><td>                   </td></tr>
+     *   <tr><td><strong> PowerPoint </strong></td><td> Y                 </td><td>                   </td></tr>
+     *   <tr><td><strong> Word       </strong></td><td> Y                 </td><td>  Y                </td></tr>
```

Comment:
To improve accessibility, consider changing "Y" values to something like "Supported".